code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2262,3 +2262,28 @@
 	color = "#623301"
 	taste_mult = 1.2
 
+/datum/reagent/eldritch
+	name = "Eldritch Essence"
+	description = "Strange liquid that defies the laws of physics"
+	taste_description = "Ag'hsj'saje'sh"
+	color = "#1f8016"
+
+/datum/reagent/eldritch/on_mob_life(mob/living/carbon/M)
+	if(isecultist(M))
+		M.drowsyness = max(M.drowsyness-5, 0)
+		M.AdjustAllImmobility(-40, FALSE)
+		M.adjustStaminaLoss(-10, 0)
+		M.adjustToxLoss(-2, 0)
```

Comment:
it's kind of funny that, even though flesh cultists are supposed to be the cultists most closely attuned to life, rust cultists are the ones with access to the best/most regeneration abilities

then again, rust cultists are the ones who are supposed to be going loud/shanking people personally, so it makes sense that they'd have good regen to back that playstyle up (in other words, this is fine)

Reply:
i prioritized gameplay in this scenario

Reply:
ye, understandable